tox.ini

```diff
@@ -41,16 +41,18 @@ deps =
     pytest-cov
     neuron-nightly
     morphio
+    -r tests/unit-mpi/requirements.txt
```

Comment:
LGTM in general. Just a nitpick: Is it better to rename [testenv:unit-ngv] to [testenv:unit-ngv-mpi] ?  So that it is clearly that the ngv tests are running with mpi.

Reply:
The same in the workflow step name.

Reply:
not all of them are with mpi... dunno. As you want